### PR TITLE
Feature/leaf 719 edit properties

### PR DESCRIPTION
**Pull request by dastan-rahimi**

### CODE REVIEW

File: LEAF_Request_Portal/admin/templates/mod_form.tpl [hunk 1]
```diff
@@ -57,13 +57,18 @@ function openContent(url) {
                                  <td>Sort Priority</td>\
                                  <td><input id="sort" type="number"></input></td>\
                              </tr>\
+														 <tr class="isSubForm">\
+																 <td>Parallel Processing <img src="../../libs/dynicons/?img=emblem-notice.svg&w=16" title="When Enabled form will become Multi-assignable"></td>\
+																 <td><select id="parallelProcessing"><option value="1">Enabled</option><option value="0">Disabled</option></select></td>\
+														 </tr>\
```
Comment (jeremyrl) [inline LEAF_Request_Portal/admin/templates/mod_form.tpl]: Spacing

File: LEAF_Request_Portal/admin/templates/mod_form.tpl [hunk 2]
```diff
@@ -161,6 +166,17 @@ function openContent(url) {
                         if(res != null) {
                         }
                     }
+                }),
+								$.ajax({
```
Comment (jeremyrl) [inline LEAF_Request_Portal/admin/templates/mod_form.tpl]: spacing

File: test/LEAF_Request_Portal_Tests/tests/api/FormEditorControllerTest.php [hunk 3]
```diff
@@ -897,4 +897,25 @@ public function testIndicatorPrivileges_setPrivileges_invalidInput() : void
...
+     * Tests the `formEditor/[digit]/formParallelProcessing` endpoint.
+     */
+    public function testSetFormParallelProcessing() : void
+    {
+        $category = self::$client->get(array('a' => 'formStack/categoryList/all'))[1];
+        $this->assertNotNull($category);
+        $this->assertEquals('form_f4687', $category['categoryID']);
+        $this->assertEquals('1', $category['parallelProcessing']);
+
+        self::$client->post(array('a' => 'formEditor/formparallelProcessing'), array(
```
Comment (jeanmarcablevets) [inline test/LEAF_Request_Portal_Tests/tests/api/FormEditorControllerTest.php]: Typo: endpoint should be formEditor/formParallelProcessing

File: test/LEAF_Request_Portal_Tests/tests/api/FormEditorControllerTest.php [hunk 4]
```diff
@@ -897,4 +897,25 @@ public function testIndicatorPrivileges_setPrivileges_invalidInput() : void
...
+
+    /**
+     * Tests the `formEditor/[digit]/formParallelProcessing` endpoint.
+     */
+    public function testSetFormParallelProcessing() : void
+    {
+        $category = self::$client->get(array('a' => 'formStack/categoryList/all'))[1];
+        $this->assertNotNull($category);
+        $this->assertEquals('form_f4687', $category['categoryID']);
+        $this->assertEquals('1', $category['parallelProcessing']);
```
Comment (jeanmarcablevets) [inline test/LEAF_Request_Portal_Tests/tests/api/FormEditorControllerTest.php]: Typo: should be `assertEquals('0', $category['parallelProcessing']);`

Comment (dastan-rahimi) [reply test/LEAF_Request_Portal_Tests/tests/api/FormEditorControllerTest.php]: default is 0 i'm enabling it

**Pull request merged**